package/MDAnalysis/analysis/encore/clustering/ClusteringMethod.py

```diff
@@ -86,8 +86,8 @@ def __call__(self, x):
 
         Returns
         -------
-        numpy.array
-            list of cluster indices
+         This method doesn't return anything, it raises a NotImplementedError
```

Comment:
Exception should be documented under the `Raises` section:
```
Raises
------
NotImplementedError
   EXPLANTION OF WHEN THIS EXCEPTION OCCOURS
```

You can familiarize yourself with the different [numpydoc sections](https://numpydoc.readthedocs.io/en/latest/format.html#sections) and the [numpydoc docstring guide](https://numpydoc.readthedocs.io/en/latest/format.html)

Reply:
Thanks:)